ui/perfherder/graphs/SelectedTestsContainer.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { react2angular } from 'react2angular/index.es2015';
+import { Container } from 'reactstrap';
+
+import perf from '../../js/perf';
+
+import TestCard from './TestCard';
+
+const SelectedTestsContainer = props => {
+  // TODO seriesList is the same as testsDisplayed in TestDataModel - change
+  // name to keep it consistent
+  const { seriesList } = props;
+  return (
```

Comment:
I usually try to follow this eslint rule, even though we don't have it activated: https://eslint.org/docs/rules/newline-after-var

Up to you, though.  :)

Reply:
That's a great rule - thanks!